### PR TITLE
Replace oracle for UNIV2 token

### PR DESCRIPTION
**Pull request by gbalabasquer**
# Description

Spell Description:

COB Team Name or Author(s):

# Contribution Checklist

- [ ] First commit title starts with COB Team Name and Collateral type (e.g. `___-TOKEN-X`)
- [ ] Code approved
- [ ] Tests approved
- [ ] CI Tests pass

# Checklist

- [ ] Every contract variable/method declared as public/external private/internal
- [ ] Verify expiration (`4 days + 2 hours` monthly and `30 days` for the rest)
- [ ] Verify hash in the description matches [here](https://emn178.github.io/online-tools/keccak_256.html)
- [ ] Validate all addresses used are in Kovan changelog
- [ ] Deploy spell to kovan `SOLC_FLAGS="--optimize --optimize-runs=1" dapp --use solc:0.6.11 build --extract && dapp create DssSpell --gas=5000000 --gas-price="$(seth --to-wei 3 "gwei")"`
- [ ] Ensure contract is verified on `kovan` etherscan
- [ ] Change test to use kovan spell address and deploy timestamp
- [ ] Keep `Kovan-DssSpell.sol` and `Kovan-DssSpell.t.sol` the same, but make a copy in `archive`
- [ ] `squash and merge` this PR

### CODE REVIEW

File: src/Kovan-DssSpell.sol [hunk 1]
```diff
@@ -19,8 +19,7 @@ pragma solidity 0.6.12;
 import {Fileable, ChainlogLike} from "dss-exec-lib/DssExecLib.sol";
 import "dss-exec-lib/DssExec.sol";
 import "dss-exec-lib/DssAction.sol";
-import "dss-interfaces/dss/ClipAbstract.sol";
-import "dss-interfaces/dss/ClipperMomAbstract.sol";
+import "dss-interfaces/dss/LPOsmAbstract.sol";
 
 struct Collateral {
```
Comment (naszam) [inline src/Kovan-DssSpell.sol:24]: This can be removed

File: src/Kovan-DssSpell.sol [hunk 2]
```diff
@@ -19,8 +19,7 @@ pragma solidity 0.6.12;
 import {Fileable, ChainlogLike} from "dss-exec-lib/DssExecLib.sol";
```
Comment (naszam) [inline src/Kovan-DssSpell.sol:19]: This can be removed

File: src/Kovan-DssSpell.sol (hunk 1 above)
Comment (gbalabasquer) [reply src/Kovan-DssSpell.sol]: Removed!

File: src/Kovan-DssSpell.sol (hunk 2 above)
Comment (gbalabasquer) [reply src/Kovan-DssSpell.sol]: Removed!

**Review by godsflaw — APPROVED** [review]

**Review by naszam — APPROVED** [review]

**Review by naszam — COMMENTED** [review]
Checked:

- dss-interfaces ok
- LP Oracle (verified, constructor args ok (src, wat, orb0, orb1), wards ok, matches github)
- Spell Actions Looks Good
- Tests Looks Good
- Deployed Spell Verified
- Deployed Code Matches GitHub
- Dss-Exec-Lib v0.0.6
- addresses_kovan.sol Matches
- Archive Ok
- Test PASS (with deployed spell address)

```Haskell
dapp-test: rpc block: latest
Running 8 tests for src/Kovan-DssSpell.t.sol:DssSpellTest
[PASS] testSpellIsCast_UNIV2DAIETH_PIP_replacement() (gas: 884324)
[PASS] testOnTime() (gas: 577257)
[PASS] testFailTooEarly() (gas: 8377)
[PASS] testSpellIsCast_GENERAL() (gas: 6486061)
[PASS] testFailWrongDay() (gas: 8465)
[PASS] testCastCost() (gas: 580876)
[PASS] testNewChainlogValues() (gas: 581561)
[PASS] testFailTooLate() (gas: 8619)
```

Missing:

Medianizer _orb0 not verified:
https://kovan.etherscan.io/address/0x91A8A02393cC21CeE04B482b0e4Aa26BfA1AA79c

LGTM

**Pull request merged**